thunder/tests/test_apex_fused_norms.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import torch
+from torch.testing import assert_close
+
+fused_layer_norm_cuda = pytest.importorskip("fused_layer_norm_cuda")
+from apex.normalization.fused_layer_norm import FusedRMSNormAffineMixedDtypesFunction
+from thunder.executors.apex_fused_rms_norm import register_apex_fused_rms_norm
+from thunder.executors.apex_entropyex import apex_ex
+import thunder
+
+
+@pytest.mark.parametrize("requires_grad", [True, False])
+@pytest.mark.parametrize("memory_efficient", [True, False])
+def test_apex_fused_norm(requires_grad, memory_efficient):
```

Comment:
IMHO it'd be better if the name is e.g. `test_apex_fused_rms_norm` for clarity. what do you think?

Reply:
Agreed